packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -25,8 +25,10 @@ const String _kPath = '/runner';
 
 String shellPath;
 
-void installHook() {
-  hack.registerPlatformPlugin(<TestPlatform>[TestPlatform.vm], () => new FlutterPlatform());
+void installHook({ int observatoryPort }) {
+  hack.registerPlatformPlugin(
+      <TestPlatform>[TestPlatform.vm],
+      () => new FlutterPlatform(observatoryPort: observatoryPort));
 }
```

Comment:
Why pipe this through install hook instead of just making it a static property of FlutterPlatform ?